src/StoreKeeper/WooCommerce/B2C/Imports/OrderImport.php

```diff
@@ -42,6 +42,10 @@ public function __construct(array $settings = [])
      */
     public function run($options = [])
     {
+        if (!$this->lock()) {
+            $this->logger->error('Cannot run. lock on.');
```

Comment:
this statement is never reached because lock throws exception

Reply:
Changes implemented.